documentation/messages/user.registration.transactional.md

```diff
@@ -0,0 +1,121 @@
+# User registration transactional message
+
+User registration event.
+
+**Routing key**
+
+`user.registration.transactional`
+
+**Exchanges**
+- `transactionalExchange`
+
+**Queues**
+- `activityStatsQueue`
+- `loggingQueue`
+- `mobileCommonsQueue`
+- `transactionalQueue`
+- `userAPIRegistrationQueue`
+- `userRegistrationQueue`
+
+**Parameters**
+
+```js
+{
+
+  /* Required. Always must be `user_register`. */
+  activity: String,
+
+  /* Required. */
+  email: String,
+
+  /* Required. */
+  uid: Number,
+
+  /* Required, Example: `US`. */
+  user_country: String,
```

Comment:
Should this be based on a list of DoSomething affiliate country code? Do we describe the specific list?

https://github.com/DoSomething/mb-toolbox/blob/master/src/MB_Toolbox.php#L79-L91